capstone-rs/Cargo.toml

```diff
@@ -28,6 +28,6 @@ name = "my_benchmark"
 harness = false
 
 [features]
-default = []
-
+default = ["not_diet"]
+not_diet = ["capstone-sys/not_diet"]
```

Comment:
Can we think of a better name than `not_diet`? We need a "positive" term, but extra negatives are confusing.

Ideas:
- `full`
- `detail`

Also, please add a comment explaining what the feature does and include a link to the upstream documentation:
https://www.capstone-engine.org/diet.html